downstream/modules/platform/ref-ldap-config-on-pah.adoc

```diff
@@ -0,0 +1,161 @@
+
+* automationhub_authentication_backend
+* automationhub_ldap_server_uri
+* automationhub_ldap_bind_dn
+* automationhub_ldap_bind_password
+* automationhub_ldap_user_search_base_dn
+* automationhub_ldap_group_search_base_dn
+
+If any of these variables are missing, the Ansible Automation  installer will  not complete the installation. 
+
+If you require additional parameters for your LDAP configuration (for example, super user access, mirroring, grant or deny access, debug logging, and the like), follow instructions in link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html-single/red_hat_ansible_automation_platform_installation_guide/index#proc-configuring-extra-ldap-parameters.adoc[Configuring extra LDAP parameters].
+
+
+== Setting up your inventory file variables
+
+When you configure your {PrivateHubName} with LDAP authentication, you must set the proper variables in your inventory files during the installation process.
+
+.Prerequisites
+
+* Ensure that your system is running Red Hat Ansible Automation Platform 2.2.1 or later.
+* Ensure that you are using {PrivateHubName} 4.5.2 or later.
+
+.Procedure
+
+. Access your inventory file according to the procedure in link:hhttps://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html-single/red_hat_ansible_automation_platform_installation_guide/index#proc-editing-installer-inventory-file_platform-install-scenario[Editing the Red Hat Ansible Automation Platform installer inventory file].
+
+. Use the following example as a guide to set up your Ansible Automation Platform inventory file:
++
+-----
+automationhub_authentication_backend = "ldap"
+
+automationhub_ldap_server_uri = "ldap://ldap:389"   (for LDAPs use  automationhub_ldap_server_uri = "ldaps://ldap-server-fqdn")
+automationhub_ldap_bind_dn = "cn=admin,dc=ansible,dc=com"
+automationhub_ldap_bind_password = "GoodNewsEveryone"
+automationhub_ldap_user_search_base_dn = "ou=people,dc=ansible,dc=com"
+automationhub_ldap_group_search_base_dn = "ou=people,dc=ansible,dc=com"
+-----
++
+[Note] 
+====
+The inventory files also contain the following optional variables with default values. If required, you can update the values.
+
+-----
+auth_ldap_user_search_scope: `SUBTREE'
+auth_ldap_user_search_filter: `(uid=%(user)s)`
+auth_ldap_group_search_scope: 'SUBTREE'
+auth_ldap_group_search_filter: '(objectClass=Group)`
+auth_ldap_group_type_class: 'django_auth_ldap.config:GroupOfNamesType'
+-----
+
+====
+
+. If you plan to set up extra parameters in  your {PrivateHubName} (such as user groups, super user access, mirroring, or others), proceed to the following secton, link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html-single/red_hat_ansible_automation_platform_installation_guide/index#proc-configuring-extra-ldap-parameters.adoc[Configuring extra LDAP parameters].
```

Comment:
You can't point to an .adoc with a link because links point to the url of the published .html file. Only xrefs can point to .adoc files or preferably, the id of the file. I believe you should change this to:

link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html-single/red_hat_ansible_automation_platform_installation_guide/index#configuring-extra-ldap-parameters_platform-install-scenarios[Configuring extra LDAP parameters].

Alternatively, you could change to:

xref:configuring-extra-ldap-parameters_platform-install-scenario[Configuring extra LDAP parameters].